models/org.go

```diff
@@ -366,6 +367,44 @@ func getOwnedOrgsByUserID(sess *xorm.Session, userID int64) ([]*User, error) {
 		Find(&orgs)
 }
 
+// HasOrgVisible tells if the given user can see the given org
+func HasOrgVisible(org *User, user *User) bool {
+	// Not SignedUser
+	if user == nil {
+		if org.Visibility == VisibleTypePublic {
+			return true
+		}
+		return false
+	}
+
+	if user.IsAdmin {
+		return true
+	}
+	switch org.Visibility {
+	case VisibleTypePrivate:
```

Comment:
I think this `switch` block is confusing. Would a two case `if` be clearer?

Reply:
The alternative is:
```go
if (org.Visibility == VisibleTypePrivate && org.IsUserPartOfOrg(user.ID)) || org.Visibility == VisibleTypeLimited || org.Visibility == VisibleTypePublic {
	return true
}
return false
```
as such I prefer the switch case.

Reply:
To add to this, there are conflicting reviews on this switch case, and I am unable to satisfy all of them.

Reply:
I can imagine - the bigger the patch the more the bikeshedding... sorry

I was thinking instead of:
```go
switch org.Visibility {
	case VisibleTypePrivate:
		if org.IsUserPartOfOrg(user.ID) {
			return true
		}
	default:
		return true
	}
	return false
}
```

do this:
```go
if org.Visibility == VisibleTypePrivate {
	if org.IsUserPartOfOrg(user.ID) {
			return true
	}
	return false
}
return true
```

Reply:
Ah, perfect. I think that satisfies all the reviews. I will update to your suggestion. Thanks :)

Reply:
Done.